### DISCUSSION TITLE
Checkbox component + readonly

### DISCUSSION THREAD

**Discussion by elainelylee** [twilio-labs/paste#564] [General] [answered]
Is the read only version of the checkbox the same as the disabled version?

**Comment by serifluous** [comment, marked as answer]
We don't have a readonly variant of the checkbox. What are you looking to use it for?

**Comment by SiTaggart** [comment]
I will also add that readonly is not supported by the native HTML element for checkbox and radio types, so we decided to follow the specification on those ones unfortunately.
